index.html

```diff
@@ -8,23 +8,89 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <main class="ganz-oben">
+    <header class="ganz-oben">
       <h1>Willkommen</h1>
       <h2>zu chris_yooo's Page</h2>
-      <h3>ich bin Christian "chris_yooo" Strassow</h3>
+    </header>
+    <nav>
+      <a class="button" href="#erste-section">About-Me</a>
```

Comment:
I started the navigation in a similar manner, but you must add an ID otherwise you will be unable to press the top navigation button.